helm/flink-operator/templates/flink-operator.yaml

```diff
@@ -67,6 +67,8 @@ spec:
               value: -Dlog4j.configurationFile=/opt/flink-operator/conf/log4j2.properties
             - name: FLINK_OPERATOR_WATCH_NAMESPACES
               value: {{ join "," .Values.watchNamespaces  }}
+            - name: JAVA_OPERATOR_SDK_CHECK_CRD
```

Comment:
Then ENV doest not work. They only support java properties now.

https://github.com/java-operator-sdk/java-operator-sdk/pull/1016/files